index.php

```diff
@@ -0,0 +1,6 @@
+<?php
+session_start();
+
+if(!$_SESSION['loggedIn']) {
```

Comment:
do you need if(!isset($_SESSION['loggedIn']))

not sure though and cant test.

Reply:
unless youre checking if $_SESSION['loggedIn'] is false then i think we are good so ignore me